cmp/models.py

```diff
@@ -103,18 +102,47 @@ def __str__(self):
 
 
 class SoldierDeath(models.Model):
-    soldier = models.OneToOneField(Soldier, on_delete=models.CASCADE, primary_key=True)
-    date = models.DateField(null=True, blank=True)
-    company = models.ForeignKey(Company, blank=True, null=True, default="UNKNOWN", on_delete=models.CASCADE) 
-    cemetery = models.ForeignKey(Cemetery, blank=True, null=True, default=110, on_delete=models.CASCADE)
+    soldier  = models.OneToOneField(Soldier, on_delete=models.CASCADE, related_name='soldierdeath')
+    date     = models.DateField(null=True, blank=True)
+    company  = models.ForeignKey(Company, blank=True, null=True, default="UNKNOWN", on_delete=models.CASCADE, related_name='companies')
+    cemetery = models.ForeignKey(Cemetery, blank=True, null=True, default=110, on_delete=models.CASCADE, related_name='cemeteries')
     cwgc_id  = models.IntegerField(blank=True, null=True, unique=False, verbose_name="War Graves ID")
-
-    def __str__(self):
```

Comment:
Having a string model is ideal else what is returned is the __repr__ method which might just look like: '<SoldierDeath: SoldierDeath object (1)>' 👎👎 no context.